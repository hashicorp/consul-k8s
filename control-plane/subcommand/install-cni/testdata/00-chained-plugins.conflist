{
  "cniVersion": "0.3.1",
  "name": "k8s-pod-network",
  "plugins": [
    {
      "binDir": "/opt/multus/bin",
      "delegates": [
        {
          "cniVersion": "0.3.1",
          "name": "openshift-sdn",
          "type": "openshift-sdn"
        }
      ],
      "globalNamespaces": "default,openshift-multus,openshift-sriov-network-operator",
      "kubeconfig": "/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig",
      "logLevel": "verbose",
      "name": "multus-cni-network",
      "namespaceIsolation": true,
      "readinessindicatorfile": "/var/run/multus/cni/net.d/80-openshift-network.conf",
      "type": "multus"
    }
  ]
}
