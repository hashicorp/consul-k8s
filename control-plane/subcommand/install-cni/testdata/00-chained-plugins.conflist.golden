{
  "cniVersion": "0.3.1",
  "name": "k8s-pod-network",
  "plugins": [
    {
      "binDir": "/opt/multus/bin",
      "delegates": [
        {
          "cniVersion": "0.3.1",
          "name": "openshift-sdn",
          "type": "openshift-sdn"
        }
      ],
      "globalNamespaces": "default,openshift-multus,openshift-sriov-network-operator",
      "kubeconfig": "/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig",
      "logLevel": "verbose",
      "name": "multus-cni-network",
      "namespaceIsolation": true,
      "readinessindicatorfile": "/var/run/multus/cni/net.d/80-openshift-network.conf",
      "type": "multus"
    },
    {
      "cni_bin_dir": "/var/lib/cni/bin",
      "cni_net_dir": "/etc/kubernetes/cni/net.d",
      "kubeconfig": "ZZZ-consul-cni-kubeconfig",
      "log_level": "info",
      "multus": false,
      "name": "consul-cni",
      "type": "consul-cni"
    }
  ]
}
